log/logger.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from os import path
+import json
+from sys import stdout
+
+
+class Logger:
+    def __init__(self):
+        current_dir_path = path.dirname(__file__)
+        config_path = path.join(current_dir_path, '../confd/config/default.json')
```

Comment:
add an issue to move the config file reader to one class that will be imported by all who needs it

Reply:
issue created